vimeo-networking/src/main/java/com/vimeo/networking/model/User.java

```diff
@@ -234,6 +234,15 @@ public Connection getWatchLaterConnection() {
         }
         return null;
     }
+
+    @Nullable
+    public Connection getWatchedVideosConnection() {
+        if ((metadata != null) && (metadata.connections != null) &&
+            (metadata.connections.watchedVideos != null)) {
+            return metadata.connections.watchedVideos;
```

Comment:
Nit: Since this method could return null, this if could be simplified to 
```
if ((metadata != null) && (metadata.connections != null)) {
   return metadata.connections.watchedVideos;
}

return null;
```